app-libs/stf/src/trusted_call.rs

```diff
@@ -551,6 +566,122 @@ where
 				store_note(&delegator, self.call, vec![delegator.clone()])?;
 				Ok(())
 			},
+			TrustedCall::assets_transfer(from, to, id, amount) => {
+				let origin = ita_sgx_runtime::RuntimeOrigin::signed(from.clone());
+				std::println!("⣿STF⣿ 🔄 assets_transfer from ⣿⣿⣿ to ⣿⣿⣿ amount ⣿⣿⣿ {:?}", id);
+				ita_sgx_runtime::AssetsCall::<Runtime>::transfer {
+					id,
+					target: MultiAddress::Id(to.clone()),
+					amount,
+				}
+				.dispatch_bypass_filter(origin)
+				.map_err(|e| {
+					Self::Error::Dispatch(format!("assets_transfer error: {:?}", e.error))
+				})?;
+				store_note(&from, self.call, vec![from.clone(), to])?;
+				Ok(())
+			},
+			TrustedCall::assets_transfer_with_note(from, to, id, amount, _note) => {
```

Comment:
The note is not used yet. Has this been forgotten?

Reply:
no. the note is just a payload which will enter the note bucket and be displayed in UI. but it has no other effect on the execution

Reply:
Oh I see. In this case, we have a section of duplicate code :)